src/ripple/protocol/STObject.h

```diff
@@ -970,6 +971,27 @@ STObject::operator[](OptionaledField<T> const& of) ->
     return OptionalProxy<T>(this, of.f);
 }
 
+inline
+NetClock::time_point
+getsfCloseTime(STObject const& sle)
```

Comment:
I'm taking a close look at this one.  I really like the `[]` syntax.  But I'm not sure how far down the rabbit hole this one goes.  If I don't hit bottom pretty soon I'm coming back out...